src/plugins/saved_objects_management/public/management_section/objects_table/components/duplicate_result_flyout.tsx

```diff
@@ -47,10 +51,20 @@ export interface DuplicateResultFlyoutProps {
   failedCopies: SavedObjectsImportError[];
   successfulCopies: SavedObjectsImportSuccess[];
   onClose: () => void;
+  onCopy: (
+    savedObjects: DuplicateObject[],
+    includeReferencesDeep: boolean,
+    targetWorkspace: string,
+    targetWorkspaceName: string
+  ) => Promise<void>;
+  targetWorkspace: string;
+  useUpdatedUX: boolean;
+  targetWorkspaceDataSourceUrl: string;
```

Comment:
Nit: dataSourceUrlForTargetWorkspace

Reply:
Thanks for your suggestion, I will update the name